### ISSUE TITLE
GraphQL replace placeOrder+createMollieTransaction with setPaymentMethodOnCart+placeOrder

### ISSUE THREAD

**Issue by paales** [mollie/magento2#388]
# Describe the new feature

Having a separate `placeOrder` and `createMollieTransaction` is non-standard an I was wondering if we can't move to a more standard solution

## Describe the solution to be implemented

```graphql
mutation SetMolliePaymentMethod {
  setPaymentMethodOnCart(
    input: {
      cart_id: "1234"
      payment_method: {
        code: "mollie_methods_ideal"
        mollie_methods_ideal: {
          issuer: "ideal_ABNANL2A"
        }
      }
    }
  ) {
    cart {
      id
      selected_payment_method {
        code
        title
        mollie_meta {
          image
          __typename
        }
      }
    }
  }
}
```

```graphql
mutation PlaceMollieOrder {
  placeOrder(input: {
      cart_id: "{{cart_token}}"
  }) {
    order {
      mollie_redirect_url
      mollie_payment_token
    }
  }
}
```

**Comment by Frank-Magmodules** [comment]
HI @paales ,

Thank you for opening and reporting this issue. We have created an internal task for this suggestion and will try to come back to you as soon as possible. Thank you for your patience!

**Comment by Frank-Magmodules** [comment]
Hi @paales ,

We are happy to share that we've just released version [1.26.0](https://github.com/mollie/magento2/releases/tag/v1.26.0) with a direct implementation for the GraphQL placeOrder endpoint. We've also updated our [WIKI](https://github.com/mollie/magento2/wiki/GraphQL-integration) page for this. 

Thank you for your suggestion and addition on this, we are closing this issue now but please feel free to reopen in case you have any more thoughts on this topic.
